Tests/ClientTests/TelemetryWrapperTests.swift

```diff
@@ -11,11 +11,15 @@ class TelemetryWrapperTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
-
         Glean.shared.resetGlean(clearStores: true)
         Glean.shared.enableTestingMode()
```

Comment:
nit: Calling `resetGlean` [will call `enableTestingMode` for you](https://github.com/mozilla/glean/blob/13533c3a44fd63066e9009ea2ae80ba356b53bb2/glean-core/ios/Glean/Glean.swift#L406), this line shouldn't be necessary, but doesn't hurt anything to be here.

Reply:
Oh I see, I'll make the change!